### PR TITLE
fix: PAN-1971 add env file env variable

### PR DESCRIPTION
**Pull request by jpantos**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [X] Optimization
- [ ] Documentation Update

## Description
Adds the option to load the env file from a custom path defined by the variable PANTOS_ENV_FILE.

## Related Tickets & Documents

- Related Issue #PAN-1971
- Closes #

## QA Instructions

_Please replace this line with instructions on how to test your changes._

### CODE REVIEW

**Review by jacekv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -76,6 +76,22 @@ $ poetry install --no-root
 
 The Pantos Common project should be used as a utility library, for example as a submodule in an upstream project. After those steps, the modules can be imported directly from the Common library.
 
-### 3.1 Examples
+### 3.1 Configuration
+
+The Pantos Common library allows its configuration to be loaded multiple fold. This normally involves an environment file and a base YAML configuration file, which can be located in the following predefined paths:
```
Comment (juanmanuel-tirado) [inline README.md:81]: "to be loaded from multiple folders" sounds better to me

File: README.md [hunk 2]
```diff
@@ -76,6 +76,22 @@ $ poetry install --no-root
...
+
+```
+    cwd
```
Comment (markuslevonyak) [inline README.md:84]: `$PWD` and `$HOME` are usually used in paths for the working and home directory, respectively.

Comment (jpantos) [reply README.md:84]: We can also use ~ to indicate home

Comment (markuslevonyak) [reply README.md:84]: I'm fine with either.

**Review by markuslevonyak — APPROVED** [review]

**Review by juanmanuel-tirado — APPROVED** [review]

**Pull request merged**